src/parser/jobs/whm/modules/Lilies.tsx

```diff
@@ -183,7 +184,7 @@ export class Lilies extends CoreGauge {
 			tiers: SEVERITIES.BLOODLILY_LEFTOVER,
 			value: this.bloodLilyGauge.value,
 			why: <Trans id="whm.gauge.bloodlily.suggestions.leftover.why">
-				{<Plural value={this.bloodLilyGauge.value} one="# Blood Lily" other="# Blood Lilies" />} went unused.
+				{<Plural value={this.bloodLilyGauge.value / MISERY_COST} one="# Blood Lily" other="# Blood Lilies" />} went unused.
```

Comment:
Setting this plural value here feels a bit awkward with respect to the tiers/value props of the suggestion overall. We're sending in the full value of the blood lily gauge, with a tiers array that is effectively doing this 'divide by misery cost', but not actually using the MISERY_COST constant, which could lead to weird behavior in the future if the cost of spending a Misery changes. Since we're always showing a severity of MAJOR if we hit this anyways, I think we'd be better off storing the result of this division into a const, and creating a normal Suggestion with a defined MAJOR Severity if that value is > 0, rather than using TieredSuggestion.

Reply:
Done, I changed the TieredSuggestion to a Suggestion that displays if >1 blood lily was used. I kept the plural value in as future proofing in case they change whm to have more than 1 blood lily at a time.

Reply:
I'm getting a "0.333 Blood Lilies went unused" suggestion now on your third log. Can you floor the `const unusedLilies` assignment to make sure it's always an integer value?

Reply:
Fixed